ISSUE_TEMPLATE

```diff
@@ -0,0 +1,7 @@
+Thanks for your interest in .NET Core. 
+
+Please note that this repo (dotnet/core) is not for filing product issues. Here are some repos where you can file an issue:
+ * dotnet/cli - for CLI tools and questions
+ * dotnet/corefx - for API issues and questions
+ * dotnet/coreclr - for runtime issues
+* nuget/home - for NuGet questions and issues
```

Comment:
Nit: since users will see the markdown here, I think the list should consistently have no indentation, and not inconsistently have one space indentation for the first three items and no indentation for the last item.